content/guides/embed/box-view/best-practices.md

```diff
@@ -0,0 +1,47 @@
+---
+rank: 40
```

Comment:
FYI, rank is the position in all listings that you want this item to appear. For non-index pages this should basically be  1,2,3,4, etc. For index pages, it affects the position of this category or subcategory in the parent folder. Have a look at the Q3 Master Categories tab on the spreadsheet for some numbers to use.